plugin/mouse/mouse.py

```diff
@@ -258,29 +264,34 @@ def show_cursor_helper(show):
 @ctx.action_class("user")
 class UserActions:
     def noise_trigger_pop():
-        if setting_mouse_enable_pop_stops_scroll.get() >= 1 and (
-            gaze_job or scroll_job
-        ):
-            # Allow pop to stop scroll
-            stop_scroll()
+        if actions.speech.enabled():
```

Comment:
This breaks using pop-to-click in sleep mode (adding `setting_mouse_enable_pop_click = 2` in `core/modes/sleep_mode.talon`), because of the new if statement. If speech is not enabled, your new else block only checks for pop to wake, not any of the other behavior, which could be valid in sleep mode.

You probably want to change this to:

```suggestion
        # Pop can be configured to wake Talon, in which case we don't want to process it as a click.
        if not actions.speech.enabled() and setting_mouse_enable_pop_wake.get() == 1:
            actions.speech.enable()
            return
```

The early return also reduces the complexity of the function by not adding a new level of indentation / nested ifs.

Reply:
As an aside, @nriley debugged this for a little bit and we were a bit confused by the initial behavior of the existing `user.mouse_enable_pop_click = 1` definition in `settings.talon`, because changing it to `2` doesn't enable pop to click in sleep mode. But that's because the settings block in that file, absent a `mode:` header, is implicitly scoped to non-sleep modes, so you have to add it to something like `core/modes/sleep_mode.talon`.

That probably reduces the number of people using pop in sleep mode, and we should probably address it separately, but in the meantime, we shouldn't break it if people have enabled it specifically.